src/carbonio-files-ui-common/views/components/NodeGridItemUI.tsx

```diff
@@ -36,29 +36,19 @@ const FooterGrid = styled(Container)`
 	justify-content: space-between;
 `;
 
-export const ContainerCell = styled(Container).attrs<
-	{
-		$contextualMenuActive?: boolean;
-		$disabled?: boolean;
-		$disableHover?: boolean;
-		$showPreview?: boolean;
-	},
-	{ backgroundColor?: string }
->(({ $contextualMenuActive, $disabled, theme }) => ({
-	backgroundColor:
-		($disabled && getColor('gray6.disabled', theme)) ||
-		($contextualMenuActive && getColor('gray6.hover', theme)) ||
-		undefined
-}))<{
+export const ContainerCell = styled(Container)<{
 	$contextualMenuActive?: boolean;
 	$disabled?: boolean;
 	$disableHover?: boolean;
 	$showPreview?: boolean;
 }>`
 	${HoverContainer} {
-		background-color: ${({ backgroundColor }): SimpleInterpolation => backgroundColor};
+		background-color: ${({ $disabled, $contextualMenuActive, theme }): false | undefined | string =>
```

Comment:
I think false type is probably unnecessary
```suggestion
		background-color: ${({ $disabled, $contextualMenuActive, theme }): string | undefined =>
```